copilot-cli/docs/ado-cli-reference.md

```
@@ -0,0 +1,193 @@
+
+```powershell
+# Install Azure CLI
+winget install Microsoft.AzureCLI
+
+# Add DevOps extension
+az extension add --name azure-devops
+```
+
+### Authentication
+
+**Option 1: Azure AD (Recommended for SSO environments)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)